metadata.yaml

```diff
@@ -2,6 +2,18 @@ name: content-cache-k8s
 display-name: Content-cache
 description: |
   A service providing local caches of HTTP servers and useful for building content delivery networks (CDN).
+
+  A service for caching content, built on top of [Nginx](https://www.nginx.com/),
+  configurable to cache any http or https web site. Tuning options include
+  cache storage size, maximum request size to cache and cache validity duration.
+
+  This service was developed to provide front-end caching for web sites run by
+  Canonical's IS team, and to reduce the need for third-party CDNs by providing
+  high-bandwidth access to web sites via this caching front-end. 
+  
+  Currently used for a number of services including [the Snap Store](https://snapcraft.io/store),
+  the majority of Canonical's web properties including [ubuntu.com](https://ubuntu.com) and
+  [canonical.com](https://canonical.com), and [Ubuntu Extended Security Maintenance](https://ubuntu.com/security/esm).
```

Comment:
You'll need four paragraphs as defined in https://discourse.charmhub.io/t/changes-to-charmhub-io-pages/9366. You can use Indico's metadata as reference